src/Faker/Provider/it_IT/Person.php

```diff
@@ -21,70 +21,73 @@ class Person extends \Faker\Provider\Person
         '{{firstNameFemale}} {{lastName}}',
         '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
     );
-
+    
+    /**
+     * Most common italian male firstname
+     * @link https://github.com/Sanfra1407/Gente-italiana/blob/master/male-names.txt
```

Comment:
This is cheating. You're republishing a list from somewhere else under MIT. You don't have the right to do that, and Faker can't support that either.

Reply:
Ok, I think it's time to close the PR. I've just wasted my time thinking to improve you're software. People names under MIT? I'm really speechless.

Reply:
I am very disappointed as well on this reaction. Since when Names should have a license? Are you serious?

Reply:
@fzaninotto I'm not cheating: you requested me a Italian names list under a MIT license and I gave it to you. I made my own list with some names; I'm not republishing nothing. The ugly truth is that names cannot be under a license, this is ridiculous. If you wanted to have an obsolete IT_it provider this has been the right way.